incubator/etcd/values.yaml

```diff
@@ -22,3 +22,10 @@ Storage: "1Gi"
 ##   GKE, AWS & OpenStack)
 ##
 # storageClass: "-"
+
+## Node labels and tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+nodeSelector: {}
+tolerations: []
+affinity: {}
```

Comment:
In general, we want values to start with a lower case letter. However, this chart probably predates this best practice. It would be good to fix that. Until this happens, we value consistency higher than the best practice. So, please follow the existing style.

Reply:
@unguiculus fixed